### PR TITLE
Fix Golang not running inside docker

### PR DESCRIPTION
**Pull request by ItsCosmas**
This new fix includes go functionality previously removed because of this issue https://github.com/paulonteri/remote-code-execution-environment/issues/12.

This fix is as described here https://github.com/paulonteri/remote-code-execution-environment/issues/12#issuecomment-660373985

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,6 +16,9 @@ RUN yarn --production
 
 COPY ./core .
 
+ENV NODE_ENV="production"
+ENV PORT=8080
```
Comment (paulonteri) [inline Dockerfile]: Kindly remove this ENV varaibles as I now set them in the server.

**Review by paulonteri — CHANGES_REQUESTED** [review]
Kindly remove the ENV variables from the Docker container as I now set them in the server

**Comment by ItsCosmas** [comment]
I have made the requested changes here's the commit https://github.com/ItsCosmas/remote-code-execution-environment/commit/a1b63f88526b8e7dbf8db1a62c9c0891e15b185a

**Review by paulonteri — APPROVED** [review]

**Comment by paulonteri** [comment]
Thanks!

**Pull request merged**